### PR TITLE
Show activity from people i follow

### LINKED ISSUE #543: Show activity from people I'm following
A feed of activity from people I'm following should show on the homepage (and/or some other useful place) when I'm logged in.

(Obviously this needs considerable design/discussion.)

### LINKED ISSUE #2035: Don't show Unknown dates on memebr activity
They end up first on the list

### PR DESCRIPTION
**Pull request by Br3nda**
closes #543
Fixes #2035

### CODE REVIEW

**Comment by Br3nda** [comment]
We could iterate on this and have it record if the event had been seen yet, and/or email about activity
but i didn't want to go too far into building "facebook for your veges".

File: app/views/photos/_description.haml [hunk 1]
```diff
@@ -0,0 +1 @@
+took a photo #{link_to event_model.title, event_model}
```
Comment (CloCkWeRX) [inline app/views/photos/_description.haml]: i18n?

File: app/views/timeline/index.html.haml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+%h1 Timeline
```
Comment (CloCkWeRX) [inline app/views/timeline/index.html.haml]: Also i18n, along with the Following header lower down.

**Comment by Br3nda** [comment]
@CloCkWeRX do you know how to do the inline links in translations? I'm trying several ways but just getting these:
image

**Comment by CloCkWeRX** [comment]
https://stackoverflow.com/questions/23001268/haml-link-to-helper-and-i18n-in-the-same-sentence perhaps?

```
  view_more_details_html: "View more %{link} including physicians, locations, directions, documents, and more."
  link: "clinic information"
```
and
```
%p
  = I18n.t('view_more_details_html', link: link_to(I18n.t('link'), clinic))
```
?

**Comment by Br3nda** [comment]
Doesn't seem to work when the link text is something your don't want translated (the crop.name or the post.subject)

**Pull request merged**